### PR TITLE
Add un/subscribe commands to `aio::ConnectionManager`.

### PR DESCRIPTION
**Pull request by nihohit**
The last commit of this PR isn't strictly required for the PR, but it improves our test coverage of the connection manager, which is somewhat related.

### CODE REVIEW

**Comment by altanozlu** [comment]
I really liked new macro and wrapper, nice.

File: redis/src/aio/connection_manager.rs [hunk 1]
```diff
@@ -386,6 +386,44 @@ impl ConnectionManager {
         result
     }
 
+    /// Subscribes to a new channel.
+    /// It should be noted that the subscription will be removed on a disconnect, and must the re-subscribed.
```
Comment (jaymell) [inline redis/src/aio/connection_manager.rs:390]:
```suggestion
    /// It should be noted that the subscription will be removed on a disconnect and must be re-subscribed.
```

File: redis/src/aio/connection_manager.rs [hunk 2]
```diff
@@ -386,6 +386,44 @@ impl ConnectionManager {
...
+    pub async fn unsubscribe(&mut self, channel_name: impl ToRedisArgs) -> RedisResult<()> {
+        check_resp3!(self.client.connection_info.redis.protocol);
+        let mut cmd = cmd("UNSUBSCRIBE");
+        cmd.arg(channel_name);
+        cmd.query_async(self).await?;
+        Ok(())
+    }
+
+    /// Subscribes to a new channel with pattern.
+    /// It should be noted that the subscription will be removed on a disconnect, and must the re-subscribed.
```
Comment (jaymell) [inline redis/src/aio/connection_manager.rs:409]:
```suggestion
    /// It should be noted that the subscription will be removed on a disconnect and must be re-subscribed.
```

File: redis/tests/test_async.rs
Comment (jaymell) [inline redis/tests/test_async.rs:1]: Can we make sure that commit messages don't go past 72 characters per line? :nerd_face:

**Review by jaymell — APPROVED** [review]

**Pull request merged**